renku/ui/service/views/error_handlers.py

```diff
@@ -251,25 +251,27 @@ def handle_templates_create_errors(f):
     def decorated_function(*args, **kwargs):
         """Represents decorated function."""
 
-        def match_schema(target, message):
-            if re.match(f".*{target}.*contains.*unsupported characters.*", message):
-                return True
-            return False
+        def get_schema_error_message(e):
+            if getattr(e, "messages", None) and e.messages.get("_schema"):
```

Comment:
```suggestion
            if isinstance(getattr(e, "messages", None), dict) and e.messages.get("_schema"):
```
to not risk "AttributeError: e.message has not attribute `get`"